extension/src/experiments/columns/model.ts

```diff
@@ -45,12 +45,21 @@ export class ColumnsModel extends PathSelectionModel<Column> {
   }
 
   public getFirstThreeColumnOrder(): string[] {
-    return this.columnOrderState
-      .filter(
-        path =>
-          this.status[path] && this.status[path] === 2 && path !== 'Created'
-      )
-      .slice(0, 3)
+    const metrics: string[] = []
+    const params: string[] = []
+
+    for (const path of this.columnOrderState) {
+      if (this.status[path] !== 2) {
```

Comment:
[I] We should always use the enum instead of the raw value (`Status.SELECTED`). The reason being it took me a few seconds to work out what this is doing.